content/influxdb/cloud/security/secrets/manage-secrets/add.md

```diff
@@ -8,7 +8,21 @@ menu:
 weight: 301
 ---
 
-Add secrets using the `influx` command line interface (CLI) or the InfluxDB API.
+Add secrets using the {{< cloud-name "short" >}} UI, `influx` command line interface (CLI) or the InfluxDB API.
+
+## Add a secret using the InfluxDB Cloud UI
+
+1. In the {{< cloud-name "short" >}} UI, click **Settings**.
+
+    {{< nav-icon "settings" >}}
+2. Click the **Secrets** tab.
+3. Click **Add Secret**.
+4. Enter a *key* and a *value*.
+   {{% warn %}}
+If will need it again, be sure to safely store your secret value.
```

Comment:
i think you can remove this line. it's confusing IMO because using a secrets manage IS a way to safely store your secret value :)